.nuspec/Microsoft.Maui.Core.targets

```diff
@@ -1,13 +1,14 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Target Name="_RemoveIncompatibleXamlFiles" BeforeTargets="BeforeBuild" Condition="$(TargetFramework.Contains('-windows')) == true and '$(SingleProject)' != 'true'">
-    <ItemGroup Condition="$(OutputType) != 'WinExe'">
-      <Page Remove="**/*.xaml">
-      </Page>
+  <Target Name="_RemoveIncompatibleXamlFiles" BeforeTargets="BeforeBuild" Condition="'$(MauiRemoveIncompatibleXaml)' != 'false' and $(TargetFramework.Contains('-windows')) == true and '$(SingleProject)' != 'true'">
```

Comment:
Added a property for a customer to disable this target.

This can either be resolved using the `$(SingleProject) == 'true'` or this new property `$(MauiRemoveIncompatibleXaml) == 'false'`. If using MauiRemoveIncompatibleXaml, then all xaml is now both Windows xaml and maui xaml. The user must now manually remove each xaml file from each respective group (MauiXaml or Page)